app/views/steps/conviction/compensation_payment_receipt/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% title t('.page_title') %>
+
+<div class="govuk-width-container">
+  <%= step_header %>
+
+  <main id="main-content" class="govuk-main-wrapper">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= error_summary %>
+        <%= step_subsection %>
+
+        <!-- The header is part of the radios legend by default, but can be configured -->
+
+        <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+
+        <div class="govuk-hint">
```

Comment:
You can probably set the hint through the locales and, like in the other view (compensation_paid_amount), set the legend to be also the heading, removing the `<h1>` too.
Take a look at the `hint` section in the helpers.yml file.